### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by acocac**
The PR proposes a temptative schedule for the challenge. We should add who is chairing in each session.

### CODE REVIEW

**Comment by acocac** [comment]
@annefou I just drafted a schedule. Let me know if it's better to display it in a sort of table? I can discuss it with Andrew H and M too.

File: challenge/details/timeline.md [hunk 1]
```diff
@@ -9,4 +9,106 @@
...
+This schedule is still very much under development.
+```
+
+All times listed below are UTC + 1 (British Summer Time). You might want to consult this [Time Zone Map](https://www.timeanddate.com/time/map/) to figure out times in your location.
+
+We suggest teams to allocated between 2 to 4 hours per day for preparing their notebooks. 
+
+# Challenge Start Day 1: May 1
+
+## 9:00 to 10:00 AM	Introductory Session
```
Comment (annefou) [inline challenge/details/timeline.md:26]:
```suggestion
## 9:00 to 10:00 AM	Introductory Session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Introductory+Session&iso=20230501T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 2]
```diff
@@ -9,4 +9,106 @@
...
+* Overview of the challenge
+* Logistics
+* Code of Conduct
+* Meet your team
+* Q & A
+
+## 10:00 to 12:00 PM Technology overview
```
Comment (annefou) [inline challenge/details/timeline.md:33]:
```suggestion
## 10:00 to 12:00 PM Technology overview ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Technology+overview&iso=20230501T10&p1=136&ah=2))
```

File: challenge/details/timeline.md [hunk 3]
```diff
@@ -9,4 +9,106 @@
...
+* Connecting to our shared JupyterHub environment
+* Git/GitHub review
+* EDS book
+
+## 4:00 to 5:00 PM Q & A
```
Comment (annefou) [inline challenge/details/timeline.md:38]:
```suggestion
## 4:00 to 5:00 PM Q & A ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Q+A&iso=20230501T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 4]
```diff
@@ -9,4 +9,106 @@
...
+* Meeting with project teams to get questions answered
+
+# Checkpoint Day 5: May 5
+## 9:00 to 10:00 AM	Morning session
```
Comment (annefou) [inline challenge/details/timeline.md:42]:
```suggestion
## 9:00 to 10:00 AM	Morning session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Morning+session&iso=20230505T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 5]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:46]:
```suggestion
## 4:00 to 5:00 PM Afternoon session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Afternoon+session&iso=20230505T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 6]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+# Chat with the experts Day 8: May 8
+## 9:00 to 10:00 AM	Morning session
```
Comment (annefou) [inline challenge/details/timeline.md:51]:
```suggestion
## 9:00 to 10:00 AM	Morning session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=JuliaGeo+Morning+session&iso=20230508T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 7]
```diff
@@ -9,4 +9,106 @@
...
+* Julia - JuliaGeo
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:55]:
```suggestion
## 4:00 to 5:00 PM Afternoon session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=JuliaGeo+Afternoon+session&iso=20230508T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 8]
```diff
@@ -9,4 +9,106 @@
...
+* Python - Pangeo/Pythia
+* Q&A
+
+# Checkpoint Day 12: May 12
+## 9:00 to 10:00 AM	Morning session
```
Comment (annefou) [inline challenge/details/timeline.md:60]:
```suggestion
## 9:00 to 10:00 AM	Morning session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Pythia+Morning+session&iso=20230512T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 9]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:64]:
```suggestion
## 4:00 to 5:00 PM Afternoon session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Pythia+Afternoon+session&iso=20230512T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 10]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+# Chat with the experts Day 15: May 15
+## 9:00 to 10:00 AM	Morning session
+* R - Climate Model Output Data Analysis
```
Comment (annefou) [inline challenge/details/timeline.md:70]:
```suggestion
* R - Climate Model Output Data Analysis ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=R+Morning+session&iso=20230515T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 11]
```diff
@@ -9,4 +9,106 @@
...
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:73]:
```suggestion
## 4:00 to 5:00 PM Afternoon session Suggested change
* R - Climate Model Output Data Analysis ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=R+Afternoon+session&iso=20230515T16&p1=136&ah=1))

```

File: challenge/details/timeline.md [hunk 12]
```diff
@@ -9,4 +9,106 @@
...
+* Reproducible Research - The Turing Way
+* Q&A
+
+# Checkpoint Day 19: May 19
+## 9:00 to 10:00 AM	Morning session
```
Comment (annefou) [inline challenge/details/timeline.md:78]:
```suggestion
## 9:00 to 10:00 AM	Morning session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Checkpoint+Morning+session&iso=20230519T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 13]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:82]:
```suggestion
## 4:00 to 5:00 PM Afternoon session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Checkpoint+Afternoon+session&iso=20230519T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 14]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+# Chat with the experts Day 22: May 22
+## 9:00 to 10:00 AM	Morning session
```
Comment (annefou) [inline challenge/details/timeline.md:87]:
```suggestion
## 9:00 to 10:00 AM	Morning session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Chat+Morning+session&iso=20230522T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 15]
```diff
@@ -9,4 +9,106 @@
...
+* Research Objects - RoHub
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:91]:
```suggestion
## 4:00 to 5:00 PM Afternoon session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Chat+Afternoon+session&iso=20230522T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 16]
```diff
@@ -9,4 +9,106 @@
...
+* Open leadership - Open Life Science Programme
+* Q&A
+
+# Checkpoint Day 26: May 25
+## 9:00 to 10:00 AM	Morning session
```
Comment (annefou) [inline challenge/details/timeline.md:96]:
```suggestion
## 9:00 to 10:00 AM	Morning session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Checkpoint+Morning+session&iso=20230525T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 17]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+## 4:00 to 5:00 PM Afternoon session
```
Comment (annefou) [inline challenge/details/timeline.md:100]:
```suggestion
## 4:00 to 5:00 PM Afternoon session ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Checkpoint+Afternoon+session&iso=20230525T16&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 18]
```diff
@@ -9,4 +9,106 @@
...
+* Team report out
+* Q&A
+
+# Project Share Out Day 29: May 29
+## 3:00 to 4:00 PM Team Presentations
```
Comment (annefou) [inline challenge/details/timeline.md:105]:
```suggestion
## 3:00 to 4:00 PM Team Presentations ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Presentation+Morning+session&iso=20230529T09&p1=136&ah=1))
```

File: challenge/details/timeline.md [hunk 19]
```diff
@@ -9,4 +9,106 @@
...
+* 10 min per team
+* 5 min for Q&A
+
+## 4:00 to 5:00 PM Wrap-up plenary and Feedback form
```
Comment (annefou) [inline challenge/details/timeline.md:109]:
```suggestion
## 4:00 to 5:00 PM Wrap-up plenary and Feedback form ([time in your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Feedback+Morning+session&iso=20230529T16&p1=136&ah=1))
```

**Review by annefou — APPROVED** [review]
I think it is a very good timeline. Not sure if we should say that all the sessions are not mandatory. I tried to add the times in local timezone as I find it easier for people to understand. Thanks!!!

**Pull request merged**